src/index.tsx

```diff
@@ -14,6 +14,12 @@ const { REACT_APP_SENTRY_DSN, REACT_APP_SENTRY_ENVIRONMENT } = process.env;
 Sentry.init({
   dsn: REACT_APP_SENTRY_DSN,
   environment: REACT_APP_SENTRY_ENVIRONMENT,
+  beforeBreadcrumb(breadcrumb) {
+    if (JSON.stringify(breadcrumb.data).includes('access_token')) {
+      delete breadcrumb.data;
```

Comment:
Wondering if we should omit the whole `data` property or getting rid of only `access_token` would do the trick.

Reply:
Any update on this? Should we omit more fields or merge this as it is?